examples/technical_blogs/ock_demo_blog/getting_started.md

```diff
@@ -0,0 +1,528 @@
+Getting Started
+===
+
+## Introduction
+
+The release package is a self-contained pre-built project to demonstrate how to run OpenCL
+ and SYCL applications on the RISC-V spike simulator, including a deep neural
+ network demo that uses the VGG network.
+
+We have put together a set of guides that serve as a walkthrough for a selection
+of OpenCL and SYCL applications.
+
+The release package contains a number of different directories:
+
+- oneAPI Construction Kit(OCK) is Codeplay’s multi-target, multi-platform framework.
+    - A few examples have been added to OCK that are useful for analyzing and demonstrating
+      the performance of OCK and oneAPI on RISC-V.
+- portDNN and portBLAS network sample project that executes VGG and Resnet50 networks
+
+```Note: cuda-to-sycl-nbody is in the process of being included to the new OCK demo package```
+
+For dpc++ support, Intel oneAPI nightly releases can be downloaded from [here](https://github.com/intel/llvm/releases). A few sycl examples can be used for testing from [oneAPI-Samples](https://github.com/oneapi-src/oneAPI-samples).
+
+You can run SYCL code using the `vector-add` from oneAPI-samples and samples from
+OCK's  `examples/applications`. Both directories implement SYCL code and therefore demonstrate
+running SYCL code using the oneAPI.
+
+You can also run an implementation of the VGG and Resnet50 neural networks
+written using portDNN, SYCL and C++.
+
+## Installation and Setup
+
+OCK demo package contains the build and install directories of all components. All the repositories mentioned above are open-sourced and can be cloned as follows.
+
+- [oneAPI Construction Kit](https://github.com/codeplaysoftware/oneapi-construction-kit.git)
+- [portDNN](https://github.com/codeplaysoftware/portDNN.git)
+- [portBLAS](https://github.com/codeplaysoftware/portBLAS.git)
+- [oneAPI Samples](https://github.com/oneapi-src/oneAPI-samples)
```

Comment:
I don't think we need oneAPI Samples it's too big, and we now have our own examples